CMakeLists.txt

```diff
@@ -50,6 +51,14 @@ if(HAIKU)
   set(ASAN OFF)
 endif()
 
+if(AMIGA)
+  set(ASAN OFF)
+  set(NONET ON)
+  set(USE_SDL1 "Use SDL1.2 instead of SDL2" ON)
+  find_package(Freetype REQUIRED)
+  find_package(PNG REQUIRED)

```

Comment:
Afaik it is possible to compile SDL/SDL_ttf without PNG support.

Reply:
Yes, of course. The libs available/distributed for Amiga is however compiled with PNG support.

Reply:
```suggestion
  set(FT_WITH_ZLIB ON)
  set(FT_WITH_PNG ON)
```